fantasy_postseason_challenge/dashboard.py

```diff
@@ -51,24 +51,35 @@ def select_team(league_id):
     if request.method == "POST":
         if form.validate_week_1():
             #TODO: Update instead of create new object when team already exists.
-            fantasy_team = FantasyTeam(
-                QB = form.data["QB"],
-                RB1 = form.data["RB1"],
-                RB2 = form.data["RB2"],
-                WR1 = form.data["WR1"],
-                WR2 = form.data["WR2"],
-                TE = form.data["TE"],
-                FLEX = form.data["FLEX"],
-                K = form.data["K"],
-                D_ST = form.data["D_ST"]
-            )
-            fantasy_team.save()
-
             league = try_get_league_by_id(league_id)
-            for member in league.member_list:
-                if member.account_id == current_user.id:
-                    member.week_1_team = fantasy_team
-            league.save()
+            if league:
```

Comment:
No need to do if league:
the try_get_leauge_by_id already does error handling; if it returns, without an error, we have a valid league id